cirq-google/cirq_google/engine/engine_processor.py

```diff
@@ -110,8 +110,12 @@ def get_sampler(
         Returns:
             A `cirq.Sampler` instance (specifically a `engine_sampler.ProcessorSampler`
             that will send circuits to the Quantum Computing Service
-            when sampled.1
+            when sampled.
         """
+        processor = self._inner_processor()
+        if processor is not None:
```

Comment:
`self._inner_processor()` should never return None according to the type hint.

Reply:
Good point, although there are null checks on `_inner_processor()` in other parts of the code :) 

I added a bit of logic that initializes the sampler with the default values if `get_sampler` is improperly parameterized with one of either `run_name` or `config_alias`, and thus would raise a [ValueError](https://github.com/quantumlib/Cirq/blob/4afe4aebb36e56f0f7dac689c671454268eac0c3/cirq-google/cirq_google/engine/processor_sampler.py#L53). 

Instead, we correct the device config key for them and print out that the default value are being used. @wcourtney , WDYT? Right now the device spec being returned is always the Processor's default, but this will be useful in the future as the device spec reflects the device_config_key specified.